examples/jsm/loaders/ColladaLoader.js

```diff
@@ -1674,13 +1681,17 @@ class ColladaLoader extends Loader {
 						break;
 					case 'emission':
 						if ( parameter.color && material.emissive ) material.emissive.fromArray( parameter.color );
-						if ( parameter.texture ) material.emissiveMap = getTexture( parameter.texture );
+						if ( parameter.texture ) material.emissiveMap = getTexture( parameter.texture, sRGBEncoding );
```

Comment:
Is material.lightMap (set above) typically expected to be in sRGB color space, as well? Is it a color texture?

Reply:
A light map is RGB-valued, and in three.js, it is assumed to be in a linear, scene-referred color space. See https://github.com/mrdoob/three.js/issues/5838#issuecomment-68454839.

An HDR light map must be in a linear colorspace; an LDR light map can be in sRGB colorspace and converted to linear.

Reply:
Okay just to be clear I'm understanding that the lightmap should be set to use `sRGBEncoding` on load since it's using TextureLoader (not HDR) and the diffuse map is assumed to be sRGBEncoding, as well.

Thanks!

Reply:
I would say an LDR light map could be either -- it just depends on how it was baked.

On the other hand, I am not aware of the Collada spec addressing color space, so do what you want.